pkg/apis/apiserver/organization_unstructured_conversion.go

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
How about we move these functions to a dedicated pkg under `pkg/apiserver/conversion/` and then we rename this file to `organization.go`
And for function names how about we just drop the `Unstructured` in the function name? I think it is enough that it returns unstructured.Unstructured, and we don't have to mention them in function name also.
And we also should add `Organization` to function name, otherwise, function names will be collapsed once we have `Project`
So something like:
`ConvertToCoreV1alpha1Organization`?
WDYT @nmiculinic

Reply:
Moving the functions to dedicated package -> ok

function names -> 
  -> No to dropping unstructured, since it's much cleaner when you read the code
  -> Yes to adding Organization to the function name

so something like `ConvertToUnstructuredCoreV1alpha1Organization` ?

Reply:
Ok, that's even better :)